src/include/daos_mgmt.h

```diff
@@ -33,108 +33,6 @@ typedef struct {
 	d_rank_list_t			*mgpi_svc;
 } daos_mgmt_pool_info_t;
 
```

Comment:
should we do whatever core rebuild tests do here rather than removing it entirely? @wangdi1 ?